README.md

```diff
@@ -83,7 +83,7 @@ If you need to build Julia in an environment that does not allow access to the o
 **Note:** the build process fail badly if any of the build directory's parent directories have spaces or other shell meta-characters such as `$` or `:` in their names (this is due to a limitation in GNU make).
 
 Once it is built, you can run the `julia` executable using its full path in the directory created above (the `julia` directory), or, to run it from anywhere, either
-
+- add an alias (in `bash`: ` echo "alias julia='/path/to/install/folder/usr/bin/julia'" >> ~/.bashrc && source ~/.bashrc`), or
```

Comment:
There should probably not be a space before `echo` here. Also, when using the macOS binary, `usr` isn't part of the patch to the executable; IIRC it's `/Applications/Julia-0.5.app/Contents/Resources/julia/bin/julia`. So for generality perhaps it would be better to omit `usr` and just say `/path/to/install/folder/bin/julia`?